tests/tests.js

```diff
@@ -1,14 +1,77 @@
 const tape = require('tape');
-const request = require('supertest');
+const supertest = require('supertest');
 const server = require('../server.js');
+const request = require('request');
+
+const postPlace = require('../routes/middleware/post_place.js');
 
 tape('home route test: GET request to /', t => {
-  request(server)
+  supertest(server)
     .get('/')
     .end((err, res) => {
-      if (err) throw err;
+      t.error(err);
       t.equal(res.status, 200, 'should return status code 200');
       t.ok(res.text.includes('<div id="list-page-content">'), 'should return home page containing test string "<div id="list-page-content">"');
       t.end();
     });
 });
+
+tape('Test the event details route', t => {
+  const htmlSample = '<h1 class="text-warning eventName">FACN2</h1>';
+  let eventId = '';
+  request.get('https://nazareth-open-tourism-platform.herokuapp.com/events', (err, res) => {
+    if (err) return err;
+    const filteredResult = JSON.parse(res.body).filter((event) => event.en.name === 'FACN2');
```

Comment:
what if there is no event with name FACN2?
you don't need to get a specific event's id, just any event id.